better_profanity/better_profanity.py

```diff
@@ -53,7 +54,7 @@ def __init__(self, words=None):
 
     ## PUBLIC ##
 
-    def censor(self, text, censor_char="*"):
+    def censor(self, text, censor_char="*", middle_only=False, get_censored_words=False):
```

Comment:
I think it would be better to have a separate `get_censored_words(text)` function, as it is not obviously clear the returned result of `censor(get_censored_words=True)` is. You could share some/most of the code with `censor` function.

Reply:
That's great. You are right that it would be better to not repeat the same code. But we could also create a function that is used by both `censor` and `get_censored_words` to determine which words are profane.